provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -282,6 +282,19 @@ data:
     }
 ---
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: snat-operator-config
+  namespace: aci-containers-system
+  labels:
+    aci-containers-config-version: "{{ config.registry.configuration_version }}"
+    network-plugin: aci-containers
+data:
+    "start": "{{config.kube_config.snat_operator.portRange.start}}"
```

Comment:
Should this be: config.kube_config.snat_operator.portRange.start|json
to parse as integer (here and below)?

Reply:
Since we are not parsing it as an integer